### PR TITLE
docs: Looker SDK java example using maven setup.

### PR DESCRIPTION
**Pull request by bryans99**

### CODE REVIEW

File: examples/java/example_maven/pom.xml [hunk 1]
```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (joeldodge79) [inline examples/java/example_maven/pom.xml:1]: I *think* this `pom.xml` is the route we want to go for publish our "kotlin" package? When you have time let's sync on getting maven setup/pom.xml for the kotlin sdk and rtl packages so I can connect up release-please

**Review by jkaster — DISMISSED** [review]
LGTM.

One question: Does `maven-wrapper.jar` need to be checked in?

**Comment by jkaster** [comment]
Oh, maybe link to this from the `/examples/java/README.md` ?

**Comment by bryans99** [comment]
> LGTM.
> 
> One question: Does `maven-wrapper.jar` need to be checked in?

Yes. this is similar to the gradle wrapper. Basically it means the mvnw command can be run with installing any additional software.

**Comment by bryans99** [comment]
> Oh, maybe link to this from the `/examples/java/README.md` ?

Done

**Review by jkaster — APPROVED** [review]
LGTM

**Pull request merged**